File: src/main/java/com/bank/account/dto/ClientFullDTO.java
package com.bank.account.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ClientFullDTO {
    private String mail;
    private String nom;
    private String prenom;
    private String password;
}

File: front/src/interface/clientFullDTO.ts
export class  ClientFullDTO {
    public mail?: String;
    public nom?: String;
    public prenom?: String;
    public password?: String;
    constructor (mail: String, nom: String, prenom: String, password: String){
        this.mail = mail;
        this.nom = nom;
        this.prenom = prenom;
        this.password = password;
    }
}